src/Sponsors.js

```diff
@@ -27,12 +27,12 @@ const Sponsors = () => {
             {/* <h2> SPONSORS </h2> */}
             {/* <h1>Thank You to Our Sponsors</h1> */}
             {/* <Grid container item spacing={3} >*/}
-                {uploads && uploads.map((upload) =>(
-                    <Grid item xs="auto" key ={upload.id}>
+                {sponsors && sponsors.map((sponsor) =>(
+                    <Grid item xs="auto" key ={sponsor.sponsor_id}>
                         <Item> <img
                             height={100}
                             width={200}
-                            src = {upload.upload_location}
+                            src = {sponsor.company_url}
```

Comment:
URLs seems to load but throw a Cross-Origin Read Blocking error. CORB blocked cross-origin response

The result is that the images are not displayed